recipe/build-core.sh

```diff
@@ -1,10 +1,20 @@
 #!/bin/bash
 set -xe
 
+# export PS4='+(${BASH_SOURCE}:${LINENO}):'
+
 if [[ "$target_platform" == osx* ]]; then
   # make sure the vendored redis OSX build can find the correct toolchain. the SDKROOT is
   # also passed as we are using at least OSX 10.15 which moves the include directory out
   # of /usr/include to ${SDKROOT}/MacOSX.sdk/usr/include
+  if [[ "$target_platform" == osx-arm64 ]]; then
+    # https://github.com/conda-forge/bazel-toolchain-feedstock/issues/18
+    # delete the line from the template and the CXXFLAGS
+    export CXXFLAGS=${CXXFLAGS/-stdlib=libc++ /}
+    export LDFLAGS="$LDFLAGS -Wl,-framework,Foundation"
```

Comment:
Not sure why this is needed
```suggestion
    export LDFLAGS="$LDFLAGS -undefined dynamic_lookup -Wl,-framework,Foundation"
```